src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/PbfToRawAtlasCommand.java

```diff
@@ -0,0 +1,177 @@
+    }
+
+    @Override
+    public int execute()
+    {
+        // set up the output path from the parent class
+        final int code = super.execute();
+        if (code != 0)
+        {
+            return code;
+        }
+
+        getInputPBFs();
+        final String countryName = this.optionAndArgumentDelegate.getOptionArgument(COUNTRY_NAME)
+                .orElseThrow(AtlasShellToolsException::new);
+        this.pbfs.forEach(pbf ->
+        {
+            final PackedAtlas rawAtlas = (PackedAtlas) new RawAtlasGenerator(pbf).build();
+            final String pbfName = pbf.getName().replace(PBF_FILETYPE, "");
+            final String rawAtlasFilename = String.format("%s%s%s%s", countryName,
+                    CountryShard.COUNTRY_SHARD_SEPARATOR, pbfName, FileSuffix.ATLAS);
+            rawAtlas.setSaveSerializationFormat(AtlasSerializationFormat.PROTOBUF);
+            final Path concatenatedPath;
+            if (this.optionAndArgumentDelegate
+                    .hasOption(MultipleOutputCommand.OUTPUT_DIRECTORY_OPTION_LONG))
+            {
+                // save atlas to user specified output directory
+                concatenatedPath = Paths.get(getOutputPath().toAbsolutePath().toString(),
+                        rawAtlasFilename);
+            }
+            else
+            {
+                // save atlas in place
+                concatenatedPath = Paths.get(
+                        Paths.get(pbf.getAbsolutePath()).getParent().toAbsolutePath().toString(),
+                        rawAtlasFilename);
+            }
+            this.outputDelegate.printlnStdout(concatenatedPath.toAbsolutePath().toString());
+            final File outputFile = new File(concatenatedPath.toAbsolutePath().toString());
+            rawAtlas.save(outputFile);
+        });
+        return 0;
+    }
+
+    @Override
+    public String getCommandName()
+    {
+        return "pbf2raw";
+    }
+
+    @Override
+    public String getSimpleDescription()
+    {
+        return "generate raw Atlas file(s) from the given PBF shard(s)";
+    }
+
+    @Override
+    public void registerManualPageSections()
+    {
+        addManualPageSection("DESCRIPTION", PbfToRawAtlasCommand.class
+                .getResourceAsStream("PbfToRawAtlasDescriptionSection.txt"));
+        addManualPageSection("EXAMPLES", PbfToRawAtlasCommand.class
+                .getResourceAsStream("PbfToRawAtlasCommandExamplesSection.txt"));
+    }
+
+    @Override
+    public void registerOptionsAndArguments()
+    {
+        final Integer[] contexts = this.optionAndArgumentDelegate.getFilteredRegisteredContexts()
+                .toArray(new Integer[0]);
+        this.registerArgument(PBF_PATH_HINT, ArgumentArity.VARIADIC, ArgumentOptionality.REQUIRED,
+                contexts);
+        this.registerOptionWithRequiredArgument(COUNTRY_NAME, COUNTRY_NAME_DESCRIPTION,
+                OptionOptionality.REQUIRED, COUNTRY_NAME);
+        super.registerOptionsAndArguments();
```

Comment:
Awesome you got this right. The design currently requires you call `super` last, which is kinda annoying. I wish it was more intuitive.